packages/core-v2-sdk/src/redeemLiquidity/redeemLiquidity.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  BigNumber,
+  BigNumberish,
+  ContractReceipt,
+  ContractTransaction,
+  Overrides,
+  Signer,
+} from "ethers";
+
+// redeem liquidity after the term is mature to the underlying
+export async function redeemLiquidity(
+  amount: BigNumberish,
+  poolAddress: string,
```

Comment:
nit: consistent arg order with other SDK methods, should be `(poolAddress, amount, signer, overrides)`